### PR TITLE
Added Basic OLED Support

### PR DESCRIPTION
**Pull request by brandonrc**
Still need to include ghost lite tx oled.

### CODE REVIEW

**Comment by brandonrc** [comment]
image

**Comment by brandonrc** [comment]
image
GHOST TX LITE OLED

**Comment by brandonrc** [comment]
image
128x32 small screen (Ghost lite TX )support

**Comment by brandonrc** [comment]
https://user-images.githubusercontent.com/25802530/118370243-6fb28d00-b56c-11eb-9b00-e1953b320030.mp4

Ghost TX Full animation 👻 Only plays on ghost hardware, currently only ghost full tx.

File: src/lib/OLED/OLED.cpp [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+// https://github.com/olikraus/u8g2/wiki/u8g2setupcpp
+// https://www.winstar.com.tw/products/oled-module/graphic-oled-display/wearable-displays.html
+// Used on the Ghost TX Lite
+U8G2_SSD1306_128X32_UNIVISION_F_4W_SW_SPI u8g2(U8G2_R0, GPIO_PIN_OLED_SCK, GPIO_PIN_OLED_MOSI, GPIO_PIN_OLED_CS, GPIO_PIN_OLED_DC, GPIO_PIN_OLED_RST);
+#else
+// https://github.com/olikraus/u8g2/wiki/u8g2setupcpp
+U8G2_SH1106_128X64_NONAME_F_4W_SW_SPI u8g2(U8G2_R0, GPIO_PIN_OLED_SCK, GPIO_PIN_OLED_MOSI, GPIO_PIN_OLED_CS, GPIO_PIN_OLED_DC, GPIO_PIN_OLED_RST);
+#endif
+
+void helper(int x, int y, int size,  const unsigned char * image){
```
Comment (maybenikhil) [inline src/lib/OLED/OLED.cpp:19]: explain the x/y coords somewhere around here?

File: src/lib/OLED/OLED.cpp [hunk 2]
```diff
@@ -0,0 +1,126 @@
+#ifdef HAS_OLED
+
+#include "targets.h"
+#include "OLED.h"
+#include "POWERMGNT.h"
+#include <U8g2lib.h>
```
Comment (maybenikhil) [inline src/lib/OLED/OLED.cpp:6]: mention this packages location

**Review by maybenikhil — APPROVED** [review]
looks good!

**Review by maybenikhil — APPROVED** [review]

**Comment by mariusdp** [comment]
Can OLED be added to "DIY TX E28 ESP32"? If yes can somebody please point to some instructions. Thanks.

**Comment by mariusdp** [comment]
was thinking to a .96/i2c oled screen

**Pull request merged**